sync-core/src/main/java/com/cloudant/http/HttpConnectionFilterContext.java

```diff
@@ -0,0 +1,30 @@
+package com.cloudant.http;
+
+/**
+ * Created by tomblench on 30/03/15.
+ */
+public class HttpConnectionFilterContext {
+
+    public boolean replayRequest;
```

Comment:
Suggest `retryRequest` as replaying implies redoing the request verbatim (that is with whatever values the request filter gave last time).